proposals/0001-simd-process.md

```diff
@@ -101,6 +102,7 @@ flowchart LR
   Idea ---> Draft;
   Draft ---> Review;
   Review ---> Accepted;
+  Review ---> Living;
```

Comment:
Might want to show 
Accepted --> Implemented 

And distinguish "Implemented" from "Living". Sound like "Living" is a new terminal state, the state of being active, like a process to be followed (this doc) or ruleset that in of itself must be both actively followed and can consistently evolve.

Adding a definition of Implemented below could help distinguish the two states.